Added reasonable values for the missing [Economy] section for #439 startup exception

appurist opened the pull request:
Doesn't actually fix the null reference exception, that would require another code change, but the default Halcyon.sample.ini file should not ship with the missing section anyway.

kf6kjg commented:
I recommend adding a little more commentary, a la issue #70. Other than that, this has been long on my todo list - good to see it happen!

appurist commented:
I've added some entries for this section in PR #444 but I see now that my commit was also missing `EconomyBaseAccount` which should be added.

I could be wrong, but I think most of the others in Vin's comment above (like the UserLevel, Stipend, etc) are old options copied from OpenSim but not used in Halcyon. I think some of the comments are very specific to OpenSim concepts that don't apply here, and I don't think we should be attempting to document any other options in the sample INI file unless they are confirmed to still apply and work correctly in Halcyon.  I think the `ObjectCapacity` option should not be used from INI file, and may be ignored in some parts of the code in favor of the expected `region.xml` file option.

My immediate goal for _this_ issue is only to fix the null reference exception in this report, not to complete the documentation for all options. However if there are any glaring exceptions (like the `EconomyBaseAccount` above), we should add those as well, as soon as possible. So for now in this PR, I've removed `ObjectCapacity` from the sample file and added `EconomyBaseAccount`.

Vinhold commented:
Works for me. As long as the options for the prices and bank account UUID are in, I am good to go. That is what the website uses with the features in the simulator. Max prims is actually set with the region settings. So I have no idea why it was in the [Economy] section.

appurist commented:
I've added the `EconomyBaseAccount` entry, but I don't think there's any others that are safe to add without further investigation. That may be necessary for #70 but I don't think it's necessary for this one. Ready to go?

Vinhold commented:
Works for me. One advance at at time as it can be verified. :)
